libs/portal-integration-angular/src/lib/core/components/data-table/data-table.component.ts

```diff
@@ -245,4 +261,19 @@ export class DataTableComponent extends DataSortBase implements OnInit {
         return 'OCX_LIST_GRID_SORT.TOGGLE_BUTTON.DEFAULT_TITLE'
     }
   }
-}
\ No newline at end of file
+
+  mapSelectionToRows() {
+    this.selectedRows$ = this._selection$.pipe(
```

Comment:
Please use combineLatest to make sure the selection is also updated when the rows change
```
this.selectedRows$ = combineLatest([this._selection$, this._rows$]).pipe(map(([selectedRows, rows]) => {....}))
```